Fundamentos do Desenvolvimento Web/BLOCO_04/dia_01/SegundoPrograma.js

```diff
@@ -0,0 +1,8 @@
+let valorX = 20;
+let valorY = 50;
+
+if (valorX > valorY) {
+    console.log("O " + valorX + ", é o maior número")
+} else { 
+    console.log("O " + valorY + ", é o maior número")
+}
```

Comment:
Você esqueceu de considerar a possibilidade de os dois números serem iguais! Nesse caso, se isso acontecesse, o retorno seria o de que y é maior, o que não seria verdade. Sugeriria adicionar mais uma condicional pro caso de não ser maior nem menor, mesmo não tendo sido pedido, ou simplesmente não retornar nada:

```suggestion
if (valorX > valorY) {
    console.log("O " + valorX + ", é o maior número")
} else if { 
    console.log("O " + valorY + ", é o maior número")
}

```